stripper/ze_tyranny_v5_2k3.cfg

```diff
@@ -1,3 +1,73 @@
+;Fix level 4 boss killer attack particle not showing if the killer attack is used twice in a row
+modify:
+{
+	match:
+	{
+		"classname" ""
+		"targetname" ""
+	}
+	delete:
+	{
+		"OnCase05" "bosslvl4_killerStop8.5-1"
+	}
+	insert:
+	{
+		"OnCase05" "bosslvl4_killer,DestroyImmediately,,9.4,-1"
+	}
+}
+
+;Clip up a common stuck spot area
+add:
+{
+	"classname" "prop_dynamic"
+	"model" "models/props/cs_militia/crate_extrasmallmill.mdl"
+	"origin" "-7048 -6256 -312"
+	"angles" "0 0 0"
+	"rendermode" "10"
+	"StartDisabled" "0"
+	"solid" "6"
+}
+
+add:
+{
+	"classname" "prop_dynamic"
+	"model" "models/props/cs_militia/crate_extrasmallmill.mdl"
+	"origin" "-6872 -6248 -416"
+	"angles" "0 0 0"
+	"rendermode" "10"
+	"StartDisabled" "0"
+	"solid" "6"
+}
+
+add:
+{
+	"classname" "prop_dynamic"
+	"model" "models/props/cs_militia/crate_extrasmallmill.mdl"
+	"origin" "-6968 -6272 -408"
+	"angles" "0 0 0"
+	"rendermode" "10"
+	"StartDisabled" "0"
+	"solid" "6"
+}
+
+;Fix nuke not killing all ZMs (?). It looks like the VScript nuke should kill all of them off, but apparently 1 survived somehow according to players.
+;Just make it go off again 0.05 seconds after in case it was a perfect respawn or something and hopefully it is fixed, but it may well be a different issue I don't know of.
+
+;Kill off fail detector if map is counted as a win so every round doesnt have the '>FAIL ROUND<' message at the end when CTs swap to both teams.
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"targetname" "lvl_change"
+	}
+	insert:
+	{
+		"OnStartTouch" "Map_BrushRunScriptCodeKillT();1.051"
```

Comment:
We can take this if absolutely necessary, but do realize that you will effectively "bug" any already working nuke scenario by using this. As humans who swap to T on round end after first nuke will be killed off by this second nuke. Finding a proper cause or other workaround would be preferred here.

Reply:
No idea where or how nuke avoidance even worked (as all players could tell me was what level it happened on), so cant really do a proper fix for it other than this way. That being said, I removed the change, so ig just watch when someone does it again.